src/groups/groups.module.ts

```diff
@@ -2,9 +2,12 @@ import { Module } from '@nestjs/common';
 import { GroupsService } from './groups.service';
 import { GroupsController } from './groups.controller';
 import { GroupsRepository } from './groups.repository';
+import { MembersService } from '../member/member.service';
+import { MemberModule } from '../member/member.module';
 
 @Module({
+  imports: [MemberModule],
   controllers: [GroupsController],
-  providers: [GroupsService, GroupsRepository],
+  providers: [GroupsService, GroupsRepository, MembersService],
```

Comment:
```suggestion
  providers: [GroupsService, GroupsRepository],
```
MemberModule이 임포트돼있다면 membersService는 provider에 등록될 필요가 없지 않을까요??